pkg/platform/kube/project/client.go

```diff
@@ -148,6 +182,79 @@ func (c *Client) Get(getProjectsOptions *platform.GetProjectsOptions) ([]platfor
 	return platformProjects, nil
 }
 
+func (c *Client) syncProjectsCache() error {
+	c.projectsCache = []platform.Project{}
+
+	// get all managed namespaces
+	namespaces, err := c.platform.GetNamespaces()
+	if err != nil {
+		namespaces = []string{c.platform.ResolveDefaultNamespace("")}
```

Comment:
Why is this fallback not a part of `c.platform.GetNamespaces()` ?

Reply:
because on other places we may want it to fail if we get an error for GetNamespaces() (e.g. when we are not permitted to do so on k8s)